projects/rero/ng-core/src/lib/record/editor/object-type/object-type.component.html

```diff
@@ -29,7 +29,7 @@
 <!-- each object properties -->
 <div class="{{ getCssClass() }}">
   <ng-container *ngFor="let f of field.fieldGroup">
-    <div class="{{ getCssClass(f) }}" *ngIf="!f.hide" [id]="f.id">
+    <div class="{{ getCssClass(f) }}" *ngIf="!f.hide">
```

Comment:
I know it's for correcting a bug, but are you sure removing the ID does not break the functionality of the commit https://github.com/rero/ng-core/commit/87e51ef573d4e49c4921045b3780602a81b5caf3 ?

Reply:
I checked in rero-ils-ui editor and it seems that doesn't break the previous fix. 
The `jump to` seems to works properly but a double check must be done to confirm that.

Reply:
@zannkukai Can you confirm that removing `id` attribute does not break your changes?

Reply:
changes reversed